src/notation/internal/notationplayback.cpp

```diff
@@ -76,10 +76,12 @@ void NotationPlayback::init(INotationUndoStackPtr undoStack)
 
     m_playbackModel.load(score());
 
-    updateTotalPlayTime();
     m_playbackModel.dataChanged().onNotify(this, [this]() {
         updateTotalPlayTime();
     });
+    undoStack->stackLockedChanged().onNotify(this, [this]() {
```

Comment:
I don't like this solution.
We should not add undostack locking and playback dependency. 

We also lock the undostack in other places, does this mean that we will update the playback every time? Strange

I think we should solve the initial problem - the playback is loaded before the initialization of the master score

Reply:
Then I'll likely have to bow out for now, not being familiar with the web of current notification flows (yet)

Reply:
But from what I could tell we only unlocked the undoStack via that interface during score creation (take not is it not the UndoStack itself, but the INotationUndoStack-wrapper around it.